Makefile

```diff
@@ -37,217 +24,69 @@ git-hooks: ## Set up hooks in .git/hooks
+	docker-compose up hbase-init
 
-up: build-all up-all
-
-up-all: services exports
-
-.PHONY: services
-services: ## Bring up hbase, population, and sample exporter services
+service-aws: ##bring up aws and prepare the services.
+	docker-compose up -d aws
 	@{ \
-		export HBASE_TO_MONGO_EXPORT_VERSION=$(hbase_to_mongo_version); \
-		export AWS_DEFAULT_REGION=$(aws_default_region); \
-		export AWS_ACCESS_KEY_ID=$(aws_access_key_id); \
-		export AWS_SECRET_ACCESS_KEY=$(aws_secret_access_key); \
-		export S3_BUCKET=$(s3_bucket); \
-		export S3_MANIFEST_BUCKET=$(s3_manifest_bucket); \
-		export S3_PREFIX_FOLDER=$(s3_prefix_folder); \
-		export DATA_KEY_SERVICE_URL=$(data_key_service_url); \
-		export DATA_KEY_SERVICE_URL_SSL=$(data_key_service_url_ssl); \
-		docker-compose up -d hbase aws; \
 		while ! docker logs aws 2> /dev/null | grep -q $(S3_READY_REGEX); do \
 			echo Waiting for aws.; \
 			sleep 2; \
 		done; \
-		docker-compose up aws-init; \
-		docker-compose up -d dks-standalone-http dks-standalone-https; \
-		docker exec -i hbase hbase shell <<< "create_namespace 'claimant_advances'"; \
-		docker exec -i hbase hbase shell <<< "create_namespace 'penalties_and_deductions'"; \
-		docker exec -i hbase hbase shell <<< "create_namespace 'quartz'"; \
-		docker-compose up hbase-populate; \
 	}
+	docker-compose up aws-init
 
-export-table-unavailable:
-		docker-compose up hbase-to-mongo-export-table-unavailable
+service-dks-insecure: ## bring up dks on 8080
+	docker-compose up -d dks-standalone-http
 
-export-blocked-topic:
-		docker-compose up hbase-to-mongo-export-blocked-topic
+service-dks-secure: ## bring up secure dks on 8443
+	docker-compose up -d dks-standalone-https
 
-export-to-file:
-		docker-compose up hbase-to-mongo-export-file
+services-dks: service-dks-insecure service-dks-secure ## bring up the two dkses.
 
-export-to-directory:
-		docker-compose up hbase-to-mongo-export-directory
+services: services-dks service-hbase service-aws ## bring up dks, hbase, aws.
 
-export-to-s3:
-		docker-compose up hbase-to-mongo-export-s3
+export-table-unavailable: ## run htme with an unavailable table.
+	docker-compose up table-unavailable
 
-exports: export-table-unavailable export-blocked-topic export-to-file export-to-directory export-to-s3
+export-blocked-topic: ## run htme with a blocked topic.
+	docker-compose up blocked-topic
 
-.PHONY: restart
-restart: ## Restart hbase and other services
-	@{ \
-		export HBASE_TO_MONGO_EXPORT_VERSION=$(hbase_to_mongo_version); \
-		export AWS_DEFAULT_REGION=$(aws_default_region); \
-		export AWS_ACCESS_KEY_ID=$(aws_access_key_id); \
-		export AWS_SECRET_ACCESS_KEY=$(aws_secret_access_key); \
-		export S3_BUCKET=$(s3_bucket); \
-		export S3_MANIFEST_BUCKET=$(s3_manifest_bucket); \
-		export S3_PREFIX_FOLDER=$(s3_prefix_folder); \
-		export DATA_KEY_SERVICE_URL=$(data_key_service_url); \
-		export DATA_KEY_SERVICE_URL_SSL=$(data_key_service_url_ssl); \
-		docker-compose restart; \
-	}
-
-.PHONY: down
```

Comment:
Can you leave this in? Bringing down and destroying the containers etc prior to int test is good practice for a fresh start run